desktop/ui/src/main/java/org/datacleaner/panels/DatastorePanel.java

```diff
@@ -415,7 +415,7 @@ public static String getDescription(Datastore datastore) {
             return datastore.getDescription();
         }
         if (datastore instanceof FileDatastore) {
-            return ((FileDatastore) datastore).getFilename();
+            return ((FileDatastore) datastore).getFilename().replaceAll("\\{[^\\}]*\\}","");
```

Comment:
This deserves a comment I think :-)
Please also double-check if filename can ever be null (I don't think so, but I'm getting too used to NPEs).